spring-context/src/main/java/org/springframework/scheduling/config/ScheduledTaskRegistrar.java

```diff
@@ -122,6 +122,14 @@ public void setTriggerTasks(Map<Runnable, Trigger> triggerTasks) {
 	public void setTriggerTasksList(List<TriggerTask> triggerTasks) {
 		this.triggerTasks = triggerTasks;
 	}
+	
+	/**
+	 * Return trigger tasks as a list of {@link TriggerTask}
+	 * @since 4.2
+	 */
+	public List<TriggerTask> getTriggerTaskList() {
+		return this.triggerTasks;
```

Comment:
If you guys are happy with moving forward on an implementation for this after the discussion takes place in Jira, I am happy to rewrite the code as needed.